src/easyaccess.cpp

```diff
@@ -57,20 +59,17 @@ namespace Exiv2 {
 
     ExifData::const_iterator orientation(const ExifData& ed)
     {
-        static const char* keys[] = {
-            "Exif.Image.Orientation",
-            "Exif.Panasonic.Rotation",
-            "Exif.MinoltaCs5D.Rotation",
-            "Exif.MinoltaCs5D.Rotation2",
-            "Exif.MinoltaCs7D.Rotation",
-            "Exif.Sony1MltCsA100.Rotation",
-            "Exif.Sony1Cs.Rotation",
-            "Exif.Sony2Cs.Rotation",
-            "Exif.Sony1Cs2.Rotation",
-            "Exif.Sony2Cs2.Rotation",
-            "Exif.Sony1MltCsA100.Rotation"
-        };
-        return findMetadatum(ed, keys, EXV_COUNTOF(keys));
```

Comment:
The commit message is: "Remove findMetadatum as a range based loop can be used."

I agree with the transformation to std::array and the usage of range based loops. However, I think that now we have some duplication in all these range based loops which could be reduced by moving the common code to the `findMetadatum` function which is still existing at the top of the file.

Reply:
The problem is that the size of the array is part of the typename. Meaning a template would need to be used.

Reply:
Oh I see. Then it is fine as it is 👍 (unless you want to explore the usage of a template for the `findMetadatum` function)